antlr/SchemaParser.ts

```
@@ -643,513 +773,1478 @@ export class SchemaParser extends Parser {
+	}
+
+}
+
+export class DocumentContext extends ParserRuleContext {
+	public definitionList(): DefinitionListContext {
+		return this.getRuleContext(0, DefinitionListContext);
+	}
+	public EOF(): TerminalNode { return this.getToken(SchemaParser.EOF, 0); }
+	constructor(parent: ParserRuleContext | undefined, invokingState: number) {
+		super(parent, invokingState);
+	}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)